code.test.js

```diff
@@ -7,10 +7,11 @@ const testSearch =
     jsc.forall("array nat", function(arr) {
         if(arr.length > 0) {
             arr.sort(function(a, b) { return a - b; });
-            return binarySearch(arr, arr[0]) === 0;
+            return binarySearch(arr, arr[0]) === 0|| arr[binarySearch(arr,arr[0])]==arr[0];
```

Comment:
Don't understand this second part of the test.

Reply:
to avoid the case of each element being the same in the array such as [0, 0, 0]. It means arr[returned find index] is still equal to the arr[0] then find the target value we want because their value are the same.

Reply:
This looks to me like it would always be true, i.e. you're not really testing here.

Reply:
Sorry, I changed it.